src/include/sof/audio/tdfb/tdfb_comp.h

```diff
@@ -24,7 +24,7 @@
 #define TDFB_GENERIC	0
 #define TDFB_HIFIEP	1
 #define TDFB_HIFI3	0
-#elif XCHAL_HAVE_HIFI3 == 1
```

Comment:
This would better be in a separate commit, not related to the performance optimizations.